src/coreclr/System.Private.CoreLib/src/Internal/Runtime/InteropServices/ComponentActivator.cs

```diff
@@ -14,6 +15,14 @@ public static class ComponentActivator
         private static readonly Dictionary<string, IsolatedComponentLoadContext> s_assemblyLoadContexts = new Dictionary<string, IsolatedComponentLoadContext>(StringComparer.InvariantCulture);
         private static readonly Dictionary<IntPtr, Delegate> s_delegates = new Dictionary<IntPtr, Delegate>();
 
+        // Use a value defined in https://github.com/dotnet/runtime/blob/main/docs/design/features/host-error-codes.md
+        // To indicate the specific error when IsSupported is false
+        private const int HostFeatureDisabled = -2147450713;
```

Comment:
Nit: I prefer using the hex value for error codes. For example:
https://github.com/dotnet/runtime/blob/34aa080d7800eb80cecc107b3fd658edd03f58ce/src/libraries/Common/src/System/HResults.cs#L27